common/components/ui/Modal.jsx

```diff
@@ -0,0 +1,107 @@
+// @flow
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import closeIcon from 'assets/images/icon-x.svg';
+
+import './Modal.scss';
+
+export default class Modal extends Component {
+  static propTypes = {
+    isOpen: PropTypes.bool,
+    size: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
```

Comment:
![screenshot 2017-07-15 02 48 51](https://user-images.githubusercontent.com/26779763/28233481-38e8beac-6908-11e7-8a38-b13830df95f5.png)

this + wrapping children in <div style={{width: desired-max-w}}> seems like replicating existing behaviour, can you please re-check on your end?